src/model/event.rs

```diff
@@ -1346,35 +1346,32 @@ pub enum Event {
     ChannelDelete(ChannelDeleteEvent),
     /// The pins for a [`Channel`] have been updated.
     ///
-    /// Fires the [`Client::channel_pins_update`] event.
+    /// Fires the [`Client::on_channel_pins_update`] event.
```

Comment:
These doc changes shouldn't have been applied.
Albeit the links here were/are faulty anyway.